g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/common/CLDRConstants.java

```diff
@@ -50,7 +52,7 @@
 	public static final String ALL_CATEGORIES = "dates,numbers,plurals,measurements,currencies,dateFields";
 	public static final String LOCALE_TERRITORIES_PATH = "cldr/localedata/{0}/territories.json";
 	public static final String LOCALE_LANGUAGES_PATH = "cldr/localedata/{0}/languages.json";
-	public static final String CONTEXT_TRANSFORM_PATH = "cldr/localedata/{0}/contextTransforms.json";
+	public static final String CONTEXT_TRANSFORM_PATH = "cldr/contextTransforms/{0}/contextTransforms.json";
```

Comment:
I think change the folder name from 'contextTransforms' to 'misc' is better, since currently we only use contextTransforms data, but there are other data in misc, if we need extract those data, we can't put them in contextTransforms floder, need create new folder, so I suggest we keep the same with cldr data structure.

Reply:
fixed